<?xml version="1.0" ?>

<launch>
    <node pkg="tf"
        type="static_transform_publisher" 
        name="camera_link" args="0 0 0.1 0 0 0 link_7 camera_link 10" />
</launch>